open-metadata-implementation/access-services/digital-service/digital-service-api/pom.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project.  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <parent>
+        <artifactId>digital-service</artifactId>
+        <groupId>org.odpi.egeria</groupId>
+        <version>1.6-SNAPSHOT</version>
```

Comment:
we are at 1.7-SNAPSHOT currently. Best to pull master before pushing
.. but in any case a version should not be used here.

All of our versions are defined in the `<dependencyManagement>` section of the top level pom. So if there is a new artifact it needs adding there. Then here we omit the `<version>`